### ISSUE TITLE
Longevity failure - container failed to stop within 30 seconds

### ISSUE THREAD

**Issue by mhagen-vmware** [vmware/vic#2843]
It looks like it cleanly stopped here:
```
time="2016-10-21T06:04:35Z" level=warning msg="stopping 30a8e0d445adc046384297ba60d56d73adbcf715990ffd72858c035e99e82166 via hard power off due to: sending kill -KILL 30a8e0d445adc046384297ba60d56d73adbcf715990ffd72858c035e99e82166: ServerFaultCode: The attempted operation cannot be performed in the current state (Powered off)." 
time="2016-10-21T06:04:35Z" level=error msg="task failed: The attempted operation cannot be performed in the current state (Powered off)." 
time="2016-10-21T06:04:35Z" level=warning msg="power off 30a8e0d445adc046384297ba60d56d73adbcf715990ffd72858c035e99e82166 task skipped (state was already poweredOff)" 
time="2016-10-21T06:04:35Z" level=debug msg="Container(30a8e0d445adc046384297ba60d56d73adbcf715990ffd72858c035e99e82166) closing 0 log followers" 
time="2016-10-21T06:04:35Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/portlayer/exec.(*Container).stop:499] [10.15091649s] 30a8e0d445adc046384297ba60d56d73adbcf715990ffd72858c035e99e82166" 
time="2016-10-21T06:04:35Z" level=debug msg="[BEGIN] [github.com/vmware/vic/lib/portlayer/exec.(*Container).refresh:237] 30a8e0d445adc046384297ba60d56d73adbcf715990ffd72858c035e99e82166" 
time="2016-10-21T06:04:35Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/portlayer/exec.(*Container).refresh:237] [10.783507ms] 30a8e0d445adc046384297ba60d56d73adbcf715990ffd72858c035e99e82166" 
time="2016-10-21T06:04:36Z" level=debug msg="[BEGIN] [github.com/vmware/vic/lib/portlayer/exec.(*Container).refresh:237] 30a8e0d445adc046384297ba60d56d73adbcf715990ffd72858c035e99e82166" 
time="2016-10-21T06:04:36Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/portlayer/exec.(*Container).refresh:237] [5.466226ms] 30a8e0d445adc046384297ba60d56d73adbcf715990ffd72858c035e99e82166" 
time="2016-10-21T06:04:36Z" level=debug msg="Committing container 30a8e0d445adc046384297ba60d56d73adbcf715990ffd72858c035e99e82166 status as: Stopped" 
```

But for 30 seconds after that it still reported stopping:
```
01:04:56.699 	INFO 	Running command 'docker -H 10.17.109.155:2376 --tls inspect 30a8e0d445adc046384297ba60d56d73adbcf715990ffd72858c035e99e82166 | grep Status 2>&1'. 	
01:04:56.719 	INFO 	${out} =             "Status": "stopping",
```

Maybe a cache issue?

[14-1-Longevity-VCH-0-4978-container-logs.zip](https://github.com/vmware/vic/files/544534/14-1-Longevity-VCH-0-4978-container-logs.zip)

**Comment by anchal-agrawal** [comment]
https://ci.vmware.run/vmware/vic/6152

**Comment by chengwang86** [comment]
Where did you run this test? On local ESXi or vCenter?

**Comment by mhagen-vmware** [comment]
Longevity and stress vcenter cluster

**Comment by hickeng** [comment]
This is due to a missed state update here: https://github.com/vmware/vic/blob/21435bf3c66193573d438be824a0a5fef8918aa7/lib/portlayer/exec/container.go#L532

This is fixed by #2820
